docs/user-guides/finetuner.md

```diff
@@ -0,0 +1,185 @@
+# Fine-tune Models
+
+Although CLIP-as-service has provided you a list of pre-trained models, you can also fine-tune your models. 
+This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fine-tune models and use them in CLIP-as-service.
+
+For installation and basic usage of Finetuner, please refer to [Finetuner documentation](https://finetuner.jina.ai).
+You can also [learn more details about fine-tuning CLIP](https://finetuner.jina.ai/tasks/text-to-image/).
+
+## Prepare Training Data
+
+Finetuner accepts training data and evaluation data in the form of [`DocumentArray`](https://docarray.jina.ai/fundamentals/documentarray/).
```

Comment:
Instead of having hyperlinks for these classes and methods from other repos, its quite nice to use the [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) extension of sphinx. This line would then look a bit neater as follows:

```suggestion
Finetuner accepts training data and evaluation data in the form of {class}`~docarray.DocumentArray`.
```

You just need to specify in your conf.py extensions to include intersphinx and add your mentioned repos:
```
# intersphinx references
intersphinx_mapping = {'docarray': ('https://docarray.jina.ai/', None)}
```

Reply:
Ah yes, thanks for mentioning